### DISCUSSION TITLE
Multiple calendars for different types

### DISCUSSION THREAD

**Discussion by marsianer** [ical4j/ical4j#647] [Q&A] [answered]
Is there an advantage in using a separate calendar for each type?

![Radicale_Type](https://github.com/ical4j/ical4j/assets/4650127/5379d567-bef6-4990-917c-f8966dcf2add)

**Comment by patrickunterwegs** [comment, marked as answer]
Hey, I'd say there's no specific advantage, but if you use an app that supports more than one type (like events+tasks or journals+tasks) and you like linking them, you might want to keep those entries within the same calendar in order to keep them in one place for export or import purposes!
